applications/nrf5340_audio/app.overlay

```diff
@@ -0,0 +1,91 @@
+/ {
+    aliases {
+        red-pwm-led = &pwm_led_0;
```

Comment:
One new thing: We are no longer putting any "modules" into our application. That is, we want the modules to be reusable everywhere in NCS. Hence, this code should likely be put in nrf/modules or nrf/lib

Reply:
Moved the code into nrf/modules. What should I do with the overlay file?